Library/Homebrew/formula.rb

```diff
@@ -810,6 +815,35 @@ def test_fixtures(file)
   def install
   end
 
+  def install_dsym
+    # create a makefile for this so we can take advantage of multiple cores.
+    dsyms = []
+    mktemp do
+      File.open("Makefile", "w") do |f|
```

Comment:
Sure; you can create the string you're writing in advance and use `EOS.undent` with interpolated strings.